### PR TITLE
Update package template, remove astropy helpers.

### PR DESCRIPTION
**Pull request by samaloney**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,72 +1,48 @@
-************
```
Comment (nabobalis) [inline README.rst:1]: I feel like this should be reverted or be combined in some form.

File: azure-pipelines.yml [hunk 2]
```diff
@@ -1,36 +1,68 @@
...
-  CI_BUILD_URL: "https://dev.azure.com/sunpy/sunpy/_build/results?buildId=$(Build.BuildId)"
+  CI_BUILD_URL: "https://dev.azure.com/sunpy/radiospectra/_build/results?buildId=$(Build.BuildId)"
+  CIBW_BUILD: cp36-* cp37-* cp38-*
+  CIBW_SKIP: "*-win32 *-manylinux1_i686"
+
+resources:
+  repositories:
+  - repository: OpenAstronomy
+    type: github
+    endpoint: ?????
```
Comment (nabobalis) [inline azure-pipelines.yml:13]:
```suggestion
    endpoint: sunpy
```

**Comment by nabobalis** [comment]
Thanks for doing this @samaloney but your time should never have been wasted doing this.

File: azure-pipelines.yml (hunk 2 above)
Comment (Cadair) [reply azure-pipelines.yml:13]: This is better left and I can change it from the azure API when I activate the project.

File: LICENSE.rst [hunk 3]
```diff
@@ -0,0 +1,22 @@
+Copyright (c) 2020, SunPy Developers
```
Comment (Cadair) [inline LICENSE.rst:1]:
```suggestion
Copyright (c) 2020, The SunPy Developers
```

File: docs/index.rst [hunk 4]
```diff
@@ -1,10 +1,15 @@
...
-==========================
+--------------------------
 
 This is the documentation for radiospectra.
 
 .. toctree::
-  :maxdepth: 2
+   :maxdepth: 2
 
-  guide.rst
```
Comment (Cadair) [inline docs/index.rst:9]: This probably needs reverting / fixing.

File: tox.ini [hunk 5]
```diff
@@ -1,27 +1,63 @@
 [tox]
-envlist = py{27,36,37}-{offline,online,sunpydev},build_docs
-isolated_build=False
+envlist =
+    py{36,37}
```
Comment (Cadair) [inline tox.ini:3]:
```suggestion
    py{36,37,38}
```
?

**Comment by nabobalis** [comment]
Thanks @samaloney, I am sorry it took this long for someone to merge this.

**Pull request merged**